src/main/java/com/google/api/codegen/discovery/config/ApiaryConfigToSampleConfigConverter.java

```diff
@@ -47,24 +49,25 @@ public ApiaryConfigToSampleConfigConverter(
     this.typeNameGenerator = typeNameGenerator;
 
     methodNameComponents = new HashMap<String, List<String>>();
-    // Pull all the methodNameComponents out.
+    // Since methodNameComponents are used to generate the request type name, we
+    // produce them here for ease-of-access.
     for (Method method : methods) {
       String methodName = method.getName();
-      LinkedList<String> nameComponents = new LinkedList<>(Arrays.asList(methodName.split("\\.")));
+      LinkedList<String> nameComponents = new LinkedList<String>(Arrays.asList(methodName.split("\\.")));
```

Comment:
Errr... I tried but it still required that `Arrays.asList...` part, and it wasn't really any shorter.